api/v1alpha1/ramenconfig_types.go

```diff
@@ -86,6 +86,10 @@ type RamenConfig struct {
 
 	// Map of S3 store profiles
 	S3StoreProfiles []S3StoreProfile `json:"s3StoreProfiles,omitempty"`
+
+	// MaxConcurrentReconciles is the maximum number of concurrent Reconciles which can be run.
+	// Defaults to 1.
+	MaxConcurrentReconciles int
```

Comment:
Should we instead change this to a map of parameters, call it `parameters map[string]string?  So later on, RamenConfig struct does not change whenever we need to add a new parameter.